style.css

```diff
@@ -198,6 +198,183 @@ body {
   z-index: 2;
 }
 
+.order {
+  background: #FAE8E4;
+  padding-left: 137px;
+  width: 100%;
+}
+
+.order-images {
+  display: flex;
+  width: 50%;
+}
+
+.order-image__first {
+  transform: translateY(-28px);
+  z-index: 1;
+}
+
+.order-image__second {
+  margin-top: 27px;
+  transform: translateX(-303px);
+  z-index: 2;
+}
+
+.order-content {
+  background: #FFFFFF;
+  margin-bottom: 30px;
+  padding-left: 37px;
+  padding-right: 50px;
+  padding-top: 35px;
+  width: 488px;
+}
+
+.order__title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  margin: 0;
+}
+
+.order__subtitle {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 0.07em;
+  margin-bottom: 50px;
+  color: #000000;
+  opacity: 0.5;
+  text-transform: uppercase;
+}
+
+.order__description {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  margin-bottom: 40px;
+  color: #000000;
+  opacity: 0.5;
+}
+
+
+.color-list {
+  display: inline-flex;
+  flex-direction: row;
+  height: 38px;
+  list-style-type: none;
+  margin-bottom: 35px;
+  transform: translateX(-37px);
+}
+
+.color-list__item-1 {
+  width: 30.18px;
+  height: 30px;
+  background: #DEDEDE;
+  border-radius: 50%;
+  margin-right: 12px;
+  margin-top: 4px;
+}
+
+.color-list__item-2 {
+  width: 38px;
+  height: 38px;
+  background: #F29F9F;
```

Comment:
Codacy found an issue: [Expected "#F29F9F" to be "#f29f9f" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)